app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const getTopics = require("./controllers/topicsController");
+const app = express();
+
+app.use(express.json());
+
```

Comment:
I know why you have included express.json() here but we should only be committing code that is relevant to the PR and so this should not have been included.